internal/services/containers/container_registry_resource.go

```diff
@@ -136,10 +136,13 @@ func resourceContainerRegistry() *pluginsdk.Resource {
 				Default:  true,
 			},
 
+			// TODO 3.0 - Remove this property as all the Classic sku instances are now deprecated and out of support at the serice side.
 			"storage_account_id": {
-				Type:     pluginsdk.TypeString,
-				Optional: true,
-				ForceNew: true,
+				Type:       pluginsdk.TypeString,
+				Optional:   true,
+				Computed:   true,
+				ForceNew:   true,
+				Deprecated: "this property is not functional at all, will remove in v3.0",
```

Comment:
Deprecated: "this attribute is no longer recognized by the API and is not functional anymore, thus this property will be removed in v3.0",